pkg/console/operator/operator.go

```diff
@@ -240,7 +240,10 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 	err := c.sync_v400(updatedStatus, configs)
 	if err != nil {
 		if !customerrors.IsSyncError(err) || !customerrors.IsCustomLogoError(err) {
-			c.SyncStatus(c.ConditionResourceSyncDegraded(updatedStatus, err.Error()))
+			// TODO: getting the err type like this is not ideal, this mechanism needs rework
```

Comment:
This little mechanism has always been a bandaid.  
But, I don't want to do too much here.  Perhaps we will need to backport this fix to 4.1? I don't know if thats necessary or not, so want to keep this PR small enough that its possible should the need arise.